.storybook/preview.js

```diff
@@ -34,7 +35,7 @@ export const parameters = {
         'Claim Status Tracker',
         ['Page', '404'],
         'Component',
-        ['Page Section', ['Header', 'Main', 'Claim Section', 'Claim Status', 'Next Steps', 'Claim Details', 'Footer']],
```

Comment:
I assume this is intentional to remove Next Steps, but why doesn't it show up in the "before" screenshot?

Reply:
it was a remnant from awhile ago - it didn't do anything